maintainers/maintainer-list.nix

```diff
@@ -6512,6 +6512,11 @@
     githubId = 1817528;
     name = "Igor Polyakov";
   };
+  iquerejeta = {
```

Comment:
Please put this hunk into a separate commit with the message ``maintainers: add NAME`` and sort it alphabetically.